projects/igniteui-angular/core/src/services/public_api.ts

```diff
@@ -5,18 +5,15 @@ export * from './animation/animation';
 export * from './csv/csv-exporter';
 export * from './csv/csv-exporter-options';
 export * from './csv/char-separated-value-data';
-export { Direction as ɵDirection, DIR_DOCUMENT as ɵDIR_DOCUMENT, IgxDirectionality as ɵIgxDirectionality } from './direction/directionality';
+export * from './direction/directionality';
```

Comment:
Would appreciate if this is not undone - neither the directionality service nor its bits are meant to be public or long-lived for that matter. Those exports were re-done that way for a reason.

Reply:
@damyanpetev All are used by other entry points and need to be exported. They can be refactored to go into their own entry point, or under a different one, but they need to be exported.